build/media-suite_helper.sh

```diff
@@ -1031,8 +1031,9 @@ do_patch() {
         do_wget -c -r -q "$patch" "$patchName"
     elif [[ -f "$patch" ]]; then
         # Check if the patch is a local patch and copy it to the current dir
+        patch="$(realpath $patch)" # Resolve fullpatch
```

Comment:
Might wanna quote-surround `$patch`, or does `realpath` work with spaces?

Reply:
Honestly, I'm not exactly sure.